ibm_mq/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - IBM MQ
 
+## 3.0.0 / 2019-06-20
```

Comment:
Should we use `2.1.0` instead ? There is no breaking change, only new features.

Reply:
This is autogenerated, guess I chose the wrong tags